src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -562,6 +562,70 @@ public void testForceOutput() {
         Assert.assertTrue(actualVC2.get(1).isPolymorphicInSamples());
     }
 
+    /**
+     * This tests whether NON_REF alleles are properly removed, including multi-allelic sites
+     */
+    @Test
+    public void testForceOutputNonRef() {
+        final File input = new File(getToolTestDataDir() + "../CombineGVCFs/NA12878.AS.chr20snippet.g.vcf");
+
+        // No sites should be output
+        final File output1 = createTempFile("output", ".vcf");
+        final ArgumentsBuilder argsWithAllSites = new ArgumentsBuilder()
```

Comment:
Looks like there's one last place to rename `argsWithAllSites`

Reply:
ok